src/mawpy/utilities/common.py

```diff
@@ -47,6 +60,14 @@ def get_combined_stay(df_by_user: pd.DataFrame, threshold: float = 0.2) -> pd.Da
                         and distance between them is less than threshold, then this_stay, next_stay, next_to_next_stay
                          are merged.
     """
+    # Validate DataFrames
```

Comment:
We might not want to add additional validation checks to columns here.

1. This function is accessed by a workflow that processes multiple users from a large CSV file. Adding additional checks here might impact the time efficiency of the workflow. (It is better to check the impact before adding changes here.)
2. I would assume, df_by_user.empty check might not be required here based on how the code is structured using group_by user.